hack/tools/kapinger/go.mod

```diff
@@ -3,10 +3,12 @@ module github.com/microsoft/retina/hack/tools/kapinger
 go 1.22.5
 
 require (
+	golang.org/x/exp v0.0.0-20250128182459-e0ece0dbea4c
```

Comment:
non-blocking: I'm not a fan of the experimental library because it makes breaking changes all the time